Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm

```diff
@@ -1830,7 +1830,7 @@ static void activateSessionCleanup(NetworkSessionCocoa& session, const NetworkSe
     RetainPtr task = [sessionSet.sessionWithCredentialStorage.session webSocketTaskWithRequest:nsRequest.get()];
     
     // Although the WebSocket protocol allows full 64-bit lengths, Chrome and Firefox limit the length to 2^63 - 1
-    task.get().maximumMessageSize = 0x7FFFFFFFFFFFFFFFull;
+    task.get().maximumMessageSize = NSIntegerMax;
```

Comment:
If this because it ends up being smaller on watchOS?

Reply:
Yes, updated.